src/test/kotlin/com/curiouscreature/kotlin/math/MatrixTest.kt

```diff
@@ -153,6 +153,31 @@ class MatrixTest {
         )
     }
 
+    @Test
+    fun `inverse Mat4 of identity is identity`() {
+        assertEquals(Mat4.identity(), inverse(Mat4.identity()))
+    }
+
+    @Test
+    fun `inverse Mat4`() {
```

Comment:
Could you test or at least comment what happens with non-invertible matrices?

Reply:
Yep, just add the test. The primary purpose of this library is for practical graphics applications, so I'm not particularly interested in adding unnecessary checks or throwing exceptions.